trace-protobuf/build.rs

```diff
@@ -45,12 +45,10 @@ fn generate_protobuf() {
     //   example: the trace intake expects the name ContainerID rather than the PascalCase ContainerId
 
     config.type_attribute("Span", "#[derive(Deserialize, Serialize)]");
-    config.field_attribute("Span.service", "#[serde(default)]");
-    config.field_attribute("Span.parentID", "#[serde(default)]");
-    config.field_attribute("Span.error", "#[serde(default)]");
-    config.field_attribute("Span.metrics", "#[serde(default)]");
-    config.field_attribute("Span.meta_struct", "#[serde(default)]");
-    config.field_attribute("Span.type", "#[serde(default)]");
+    config.field_attribute(
```

Comment:
this makes it so if _any_ fields in pb.Span aren't present or are null during deserialization, they take on the default value.